src/vendor/wordpress/_wordpress.scss

```diff
@@ -111,6 +106,44 @@ $wp-button-gap: sizes.$list-inline-gap;
   }
 }
 
+/**
+ * Gutenberg block: Image
+ * Styles for images inserted via Gutenberg blocks.
+ *
+ * 1. Since we are applying `margin-top` to CMS content via our staggerItems
+ *    utility, the default `margin-bottom` value on Gutenberg image blocks is
+ *    unnecessary, so we remove it here.
+ * 2. Setting a width of 100% prevents resized and centered images from breaking
+ *    layout when placed after a right-aligned image in a post.
+ * 3. Applies an opinionated wide-image setting that allows an image element to
+ *    spill slightly outsides it's container.
+ */
+.wp-block-image {
+  margin-bottom: 0; /* 1 */
+  position: relative;
+
+  &.aligncenter {
+    width: 100%; /* 2 */
+  }
+
+  &.alignfull {
+    @media (min-width: breakpoint.$m) {
+      left: 50%;
+      margin-left: -50vw;
+      margin-right: -50vw;
+      right: 50%;
+      width: 100vw;
+    }
+  }
+
+  &.alignwide {
+    @media (min-width: breakpoint.$m) {
+      left: -6vw; /* 3 */
+      width: calc(100% + 12vw); /* 3 */
+    }
+  }
```

Comment:
The resulting width applied via this selector may seem a bit magical and opinionated. I arrived at this after surveying cloudfour.com and trying to establish a reasonable width based on what I saw being used most often in articles.